Assets/Mirror/Runtime/ClientScene.cs

```diff
@@ -763,13 +764,16 @@ internal static void ApplySpawnPayload(NetworkIdentity identity, SpawnMessage ms
 
             NetworkIdentity.spawned[msg.netId] = identity;
 
-            // objects spawned as part of initial state are started on a second pass
-            if (isSpawnFinished)
```

Comment:
this is a whole breaking change by itself, should probably be done in its own PR